portfolio/src/main/webapp/index.html

```diff
@@ -58,13 +58,27 @@ <h1 class="title">Ani Machaidze</h1>
                 </div>
             </div>
 
-            <!-- trying out fetching on this buttonm -->
-            <div id="content">
-                <h1>Random Quotes</h1>
-                <p>Click here to see a random quote:</p>
-                <button onclick="getRandomMessage()">Fetch a quote!</button>
-                <div id="quote-container"></div>
+
+
+            <div id="comments-form"> 
+                <form action="/data" method="POST">
+
+                    <p>Enter your name:</p>
+                    <textarea name="author"></textarea>
+                    <br/>
+                    <p>Enter your comment:</p>
+                    <textarea name="text"></textarea>
+                    <br/>
+
+                <input type="submit" />
```

Comment:
I think you'll want to add a `value` attribute as well for the input button to not be empty.

Reply:
Oh nevermind, I didn't see it gets the default "Submit" if you don't add anything.